### DISCUSSION TITLE
`<dialog>`

### DISCUSSION THREAD

**Discussion by Que-tin** [Devographics/surveys#158] [State of HTML Features]
For dialog boxes or other interactive components, such as a dismissible alerts, or subwindows.

### Code sample

```html
<dialog id="confirm">
	<form method="dialog">
		Are you sure?
		<button value="1">Yes</button>
		<button value="0">No</button>
	</form>
</dialog>
```

```html
<dialog open>
  <p>Dialog</p>
  <form method="dialog">
    <button>Close</button>
  </form>
</dialog>
```

### MDN URL:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

### Specification URL:

https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element

### How stable is this feature?

Stable (Widely supported for several years)

### Type of feature

HTML element

### What focus area does this feature fall under?

Interactivity

**Comment by lukewarlow** [comment]
There's discussion in openUI and potentially other places of firstly introducing a declarative way to open a modal dialog (similar to popovers), and secondly of deprecating non modal dialogs (in the sense that .show() would be deprecated and developers would be encouraged to use `<dialog popover>` instead.)

It would be good to touch on both of these ideas somehow?
